patrolling_action_client:
  ros__parameters:
    cycle_count: 3  
    waypoint1: [3.271313190460205, -3.657531261444092]
    waypoint2: [2.4051167964935303, 0.08290701359510422]
    waypoint3: [-4.635148048400879, 0.0460820198059082]
    waypoint4: [-6.4781622886657715, -3.712772846221924]
    waypoint5: [9.555916786193848, -11.72805118560791]